libethdrivers/plat_include/imx6/ethdrivers/plat/eth_plat.h

```diff
@@ -10,14 +10,14 @@
 #include <ethdrivers/imx6.h>
 #include <utils/attribute.h>
 #include <ethdrivers/raw.h>
-#include "../src/plat/imx6/enet.h"
```

Comment:
If I remember correctly we had to include this here to address a nasty include problem and make these prototypes available to other C file in our driver:
```
int enet_mdio_read(struct enet *enet, uint16_t phy, uint16_t reg);
int enet_mdio_write(struct enet *enet, uint16_t phy, uint16_t reg, uint16_t data);
```
Could you leave this in for now. I'd make this a separate issue than that should be cleaned up.

Reply:
It does seem possible to declare just those symbols at the bottom of this file:
```
int enet_mdio_read(struct enet* enet, uint16_t phy, uint16_t reg);
int enet_mdio_write(struct enet* enet, uint16_t phy, uint16_t reg, uint16_t data);
```

Adding a relative include into a private header file breaks if these public headers are installed somewhere else which breaks how I'm using the library.

Reply:
Yes, sounds feasible, I don't remember why we did not do this. Can you put removing this header here in a separate commit, so this is separate from adding the `NIC_CONFIG_NO_CLOCK_SYS` flag or adding the interface again?